run

```diff
@@ -2,4 +2,13 @@
 
 chown -R nexus:nexus /opt/sonatype
 chown -R nexus:nexus /nexus-data
-su-exec nexus /opt/sonatype/nexus/bin/nexus run
\ No newline at end of file
+if [[ -n "${ELASTICSEARCH_MAX_RESULT_WINDOW}" ]]
+then
+    echo >> /opt/sonatype/nexus/etc/fabric/elasticsearch.yml
+    echo "# https://issues.sonatype.org/browse/NEXUS-16917" \
+        >> /opt/sonatype/nexus/etc/fabric/elasticsearch.yml
+    echo "index.max_result_window: ${ELASTICSEARCH_MAX_RESULT_WINDOW}" \
+        >> /opt/sonatype/nexus/etc/fabric/elasticsearch.yml
+    echo >> /opt/sonatype/nexus/etc/fabric/elasticsearch.yml
```

Comment:
I'd just do it like this, since it is easier to read; the only downside is that the indentation inside the if block can't be kept. 

```suggestion
cat >> /opt/sonatype/nexus/etc/fabric/elasticsearch.yml <<EOL
# https://issues.sonatype.org/browse/NEXUS-16917
index.max_result_window: ${ELASTICSEARCH_MAX_RESULT_WINDOW}
EOL
```